account_move_line_mrp_info/views/account_move_line_view.xml

```diff
@@ -12,4 +12,16 @@
         </field>
     </record>
 
+    <record id="view_account_move_line_filter" model="ir.ui.view">
+        <field name="name">account.move.line.form</field>
```

Comment:
account.move.line.search

to avoid confusion

Reply:
sure! I missed that. Thanks.